### ISSUE TITLE
Missing tunnel between Farragut North and West

### ISSUE THREAD

**Issue by kchanba** [conveyal/modeify#272]
– The route skips the virtual tunnel between Farragut north and west. I tend to take about less then 1 hour

**Comment by trevorgerhardt** [comment]
Is that transfer quicker than just changing at Metro Center? I've never done it. @abyrd would we have to add special rules for that or is information like that available in GTFS and we're not parsing it now?

**Comment by abyrd** [comment]
Can you clarify what a "virtual tunnel" is? Does this mean you can leave the subway without re-paying?

**Comment by trevorgerhardt** [comment]
Yup, you cannot transfer directly there, but you can leave the station and switch and not get charged twice.

**Comment by kchanba** [comment]
[here is the route] (http://carfreeatoz.com/planner?from=11206%20Ashley%20Dr%2C%20Rockville%2C%20Maryland&to=3811%20N%20Fairfax%20Dr%2C%20Arlington%2C%20Virginia&modes=BICYCLE%2CBICYCLE_RENT%2CBUS%2CTRAINISH%2CWALK%2CCAR&start_time=7&end_time=9&days=M%E2%80%94F)

**Comment by trevorgerhardt** [comment]
I'd say this is a relatively low priority. It would save 2-3 minutes at the most and not much walking as you end up skipping only two total stops (the Metro Center transfer and McPherson Square on Orange/Silver). But "virtual transfers" should be looked into or at least verified that we can handle them appropriately.

**Comment by trevorgerhardt** [comment]
This is basically a "wontfix" at this point also, correct @abyrd / @demory ?
